packages/mobile/src/escrow/saga.ts

```diff
@@ -191,6 +200,30 @@ function* getSentPayments() {
   }
 }
 
+export function* fetchReclaimSuggestedFee(action: FetchReclaimTransactionFeeAction) {
+  try {
+    const { paymentID } = action
+    const escrow = yield call(getEscrowContract, web3)
+    const stableToken = yield call(getStableTokenContract, web3)
+    const account = yield call(getConnectedUnlockedAccount)
+
+    Logger.debug(TAG + '@fetchReclaimSuggestedFee', 'Fetching reclaim transaction fee')
+
+    const mockReclaimTx = escrow.methods.revoke(paymentID)
+    const mockTxParams: any = { from: account, gasCurrency: stableToken._address }
+    const gas = new BigNumber(yield mockReclaimTx.estimateGas(mockTxParams))
+    const gasPrice = new BigNumber(yield call(fetchGasPrice))
+    const suggestedFeeInWei = gas.multipliedBy(gasPrice)
+
+    yield put(setReclaimTransactionFee(suggestedFeeInWei.toString()))
+
+    Logger.debug(TAG + '@fetchReclaimSuggestedFee', `New reclaim tx fee is: ${suggestedFeeInWei}`)
+  } catch (e) {
+    yield put(showError(ErrorMessages.FETCH_RECLAIM_FEE_FAILED, ERROR_BANNER_DURATION))
+    throw e
```

Comment:
You resolved my comment but this throw is still here

Reply:
whoops sorry about that!